NetCore/LoadResizeSave.cs

```diff
@@ -143,6 +146,41 @@ public void SystemDrawingResize(string input)
             }
         }
 
+        [Benchmark(Description = "ImageFlow Load, Resize, Save"/*,
+            OperationsPerInvoke = ImagesCount*/)]
+        public async Task ImageFlowBenchmark()
+        {
+            foreach (string image in Images)
+            {
+                await ImageFlowResize(image);
+            }
+        }
+
+        public async Task ImageFlowResize(string input)
+        {
+            using (var output = File.Open(OutputPath(input, ImageFlow), FileMode.Create))
+            {
+                var imageBytes = File.ReadAllBytes(input);
+                // Resize it to fit a 150x150 square
+                using (var image = new ImageFlow.ImageJob())
+                {
+                    var o = 
+                        await image
+                            .Decode(imageBytes)
+                            .ResizerCommands($"width={ThumbnailSize}&height={ThumbnailSize}&mode=max")
+                            .EncodeToBytes(new ImageFlow.MozJpegEncoder(Quality, true))
+                            .Finish()
+                            .InProcessAsync();
+
+                    if (o.First.TryGetBytes().HasValue)
```

Comment:
Do we want to throw if no value exists? looks like we'd otherwise be outputting nothing, and if we don't notice, that would skew the results.